examples/java/scenarios/src/main/java/main/MintBurnTransferCustodial.java

```diff
@@ -0,0 +1,104 @@
+package main
+import com.bloxbean.cardano.client.address.Address;
+import com.bloxbean.cardano.client.common.model.Networks;
+import com.bloxbean.cardano.client.crypto.Keys;
+import com.bloxbean.cardano.client.exception.CborSerializationException;
+import com.google.protobuf.Struct;
+import com.google.protobuf.Value;
+import io.grpc.stub.StreamObserver;
+import iog.psg.client.nativeassets.NativeAssetsApi;
+import iog.psg.client.nativeassets.NativeAssetsBuilder;
+import iog.psg.client.nativeassets.multisig.v1.NativeAssetsMultisigApi;
+import iog.psg.service.nativeassets.*;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.ExecutionException;
+import java.util.function.Function;
+
+public class MintBurnTransferCustodial {
+
+    public static <E, R> StreamObserver<E> lastResult(CompletableFuture<R> result, Function<E, R> f) {
+        return new StreamObserver<>() {
+            ArrayList<R> buff = new ArrayList();
+
+            @Override
+            public void onNext(E event) {
+                R converted = f.apply(event);
+                System.out.println("Got interim response: " + converted);
+                buff.add(converted);
+            }
+
+            @Override
+            public void onError(Throwable throwable) {
+                System.out.println("Got error: " + throwable.getCause());
+            }
+
+            @Override
+            public void onCompleted() {
+                result.complete(buff.get(buff.size() - 1));
+            }
+        };
+
+    }
+
+    public static void main(String[] args) throws ExecutionException, CborSerializationException, InterruptedException {
+        String apiKey = ""; // apiKey
+        String clientId = ""; // clientId
+        // Create the Api
+        NativeAssetsApi api = NativeAssetsBuilder.create(apiKey, clientId).build();
+        NativeAssetsMultisigApi helper = NativeAssetsBuilder.create(apiKey, clientId).buildMultisig();
+
+        Keys keys = helper.generateKeys();
+        Address targetTransferAddress = helper.generateAddress(keys.getVkey(), Networks.mainnet());
```

Comment:
I'm not sure if we should include any key/address generation in custodial version examples. I'd rather use an example address or address placeholder instead to make it simpler.

Reply:
ok, reworked